pages/Transactions/index.tsx

```diff
@@ -1,8 +1,89 @@
+import * as React from 'react';
 import Layout from "../../components/Layout/Layout";
+import PropTypes from 'prop-types';
+import { Grid, Button, Tabs, Tab, Box } from "@mui/material";
+import styles from './Transactions.module.css';
 
 const TransactionsPage: React.FunctionComponent = () => {
+
+    function TabPanel(props) {
```

Comment:
While using bindings is correct, try using arrow functions moving forward! This is just to keep the code throughout our codebase consistent.